AdminWebsite/AdminWebsite/Controllers/PersonsController.cs

```diff
@@ -58,7 +58,17 @@ public async Task<ActionResult<IList<PersonResponse>>> PostPersonBySearchTerm([F
                 term = _encoder.Encode(term);
                 var searchTerm = new SearchTermRequest(term);
 
-                var personsResponse = await _bookingsApiClient.PostPersonBySearchTermAsync(searchTerm);
+                var excludedEmails = (await _userAccountService.GetJudgeUsers()).Select(x => x.Email.ToLower()).ToList();
```

Comment:
A suggestion here, can we use toLower to toLowerInvariant.
I will be honest, I had to refer back to what the difference is, basically it ignore culture specefic characters and will use lowercase in english I believe no matter the case. 

Here is a stack overflow post I found about it: https://stackoverflow.com/questions/6225808/string-tolower-and-string-tolowerinvariant